lib/sles4sap_publiccloud.pm

```diff
@@ -282,9 +289,13 @@ sub is_hana_resource_running {
         get_required_var("INSTANCE_SID") . get_required_var("INSTANCE_ID"));
```

Comment:
This is "hidden" dependency to Ansible qe-sap-deployment resource naming convention